From 315f6657373ff41fb28d1e067101ccec672e4295 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:30 -0400
Subject: [redhat] kabi: Adding symbol param_set_bool (kernel/params.c)

Message-id: <177995ba79fcc13af6b6eef8dab81bf0cec8afd1.1410510773.git.poros@redhat.com>
Patchwork-id: 96105
O-Subject: [PATCH RHEL7.1 150/233] kabi: Adding symbol param_set_bool (kernel/params.c)
Bugzilla: 1071098
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Not used directly, used by module_param internally for type bool.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/param_set_bool
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/param_set_bool
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/param_set_bool

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/param_set_bool b/redhat/kabi/kabi-module/kabi_ppc64/param_set_bool
new file mode 100644
index 0000000..6f5e344
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/param_set_bool
@@ -0,0 +1,2 @@
+#1-
+0x6b7589f4 param_set_bool vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/param_set_bool b/redhat/kabi/kabi-module/kabi_s390x/param_set_bool
new file mode 100644
index 0000000..6f5e344
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/param_set_bool
@@ -0,0 +1,2 @@
+#1-
+0x6b7589f4 param_set_bool vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/param_set_bool b/redhat/kabi/kabi-module/kabi_x86_64/param_set_bool
new file mode 100644
index 0000000..6f5e344
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/param_set_bool
@@ -0,0 +1,2 @@
+#1-
+0x6b7589f4 param_set_bool vmlinux EXPORT_SYMBOL
-- 
1.7.1